src/EditorFeatures/CSharp/Highlighting/KeywordHighlighters/CheckedExpressionHighlighter.cs

```diff
@@ -18,18 +18,7 @@ public CheckedExpressionHighlighter()
         {
         }
 
-        protected override IEnumerable<TextSpan> GetHighlights(
-            CheckedExpressionSyntax checkedExpressionSyntax, CancellationToken cancellationToken)
-        {
-            switch (checkedExpressionSyntax.Kind())
-            {
-                case SyntaxKind.CheckedExpression:
-                case SyntaxKind.UncheckedExpression:
-                    yield return checkedExpressionSyntax.Keyword.Span;
-                    break;
-                default:
-                    yield break;
-            }
-        }
+        protected override void AddHighlights(CheckedExpressionSyntax checkedExpressionSyntax, List<TextSpan> highlights, CancellationToken cancellationToken)
+            => highlights.Add(checkedExpressionSyntax.Keyword.Span);
```

Comment:
while i tried to make the changes mechanical, i couldn't help but simplify this :)